sirepo/pkcli/flash.py

```diff
@@ -5,20 +5,47 @@
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
 from __future__ import absolute_import, division, print_function
-from pykern import pkconfig
 from pykern import pkio
 from pykern.pkdebug import pkdp, pkdc
 from sirepo import mpi
 from sirepo import simulation_db
 from sirepo.template import template_common
-import py.path
 import sirepo.sim_data
 import sirepo.template.flash as template
 import subprocess
 
+_SIM_DATA = sirepo.sim_data.get_class('flash')
 
 def run_background(cfg_dir):
+    # TODO(e-carlin): there is a possible caching problem here
```

Comment:
@robnagler should we implement caching when getting files?

Reply:
not yet.

Reply:
7ec3b53a